tests/tests/control_p8

```diff
@@ -91,7 +91,6 @@ export USE_MERRA2=.true.
 export IAER=1011
 export NSTF_NAME=2,1,0,0,0
 
-export LHEATSTRG=.true.
```

Comment:
Why do you need to change the control_p8 variables? You should run the control_p8 test to make sure the results don't change.